subsystem/elevator.py

```diff
@@ -0,0 +1,79 @@
+import rev
+
+from toolkit import Subsystem
+from toolkit.motors.rev_motors import SparkMax, SparkMaxConfig
+
+import config
+
+# TODO: CHANGE WHEN ROBOT IS BUILT
+ELEVATOR_CONFIG = SparkMaxConfig(
+    0.055, 0.0, 0.01, 0.000, (-.5, .75), idle_mode=rev.CANSparkMax.IdleMode.kBrake
+)
+
+class Elevator(Subsystem):
+
+    motor_extend: SparkMax = SparkMax(
+        config.elevator_can_id, config=ELEVATOR_CONFIG, inverted=False
+    )
+
+    def __init__(self) -> None:
+        super().__init__()
+        # Absolute encoder
+        self.zeroed: bool = False
+
+    def init(self) -> None:
+        self.motor_extend.init()
+        self.encoder = self.motor_extend.abs_encoder()
+        self.motor_extend.motor.setClosedLoopRampRate(config.elevator_ramp_rate)
+
+    def set_length(self, length: float) -> None:
+        """
+        Sets the length of the elevator in meters
+
+        Args:
+            length (meters): Length in meters
+        """
+        
+        current_length = self.get_length()
+
+        if length > current_length:
+            ff = config.elevator_feed_forward
+        elif length < current_length:
+            ff = -config.elevator_feed_forward
+        else: ff = 0
```

Comment:
Sorry, I just realized while looking at this: what this variable is setting is the Arbitrary feedforward. 
It's added on top of things based on new info (think 2023 robot with the elevator and rotation.) We actually don't need to do this here, and use normal feedforward in the `SparkMaxConfig` class. It's the fourth parameter, so for now comment the arb FF out, and add a variable in the config for `elevator FF`